nbbuild/build.xml

```diff
@@ -470,6 +470,24 @@ Hg ID:    ${hg.id}
     </nbmerge>
   </target>
 
+  <target name="build-basic" description="Build the NetBeans Basic cluster.">
```

Comment:
Is there any reason for the `target`s to be in `nbbuild/build.xml`? I preferred when they were only in `java/build.xml`.

Reply:
To be able to call target "build-basic" from .travis.yml.
Not sure travis will accept something like  OPTS="-f java/build.xml"

Reply:
hi @JaroslavTulach , Can i merge the PR. there is an unresolved conversation?